fun_api/serializers.py

```diff
@@ -0,0 +1,17 @@
+from rest_framework.authtoken.serializers import AuthTokenSerializer
+from rest_framework import serializers
+
+from django.utils.translation import ugettext_lazy as _
+
+
+class FunAuthTokenSerializer(AuthTokenSerializer):
+
+    def validate(self, attrs):
+        super(FunAuthTokenSerializer, self).validate(attrs)
+        user = attrs['user']
+        if user.is_superuser:
+            return attrs
+        if not user.is_staff:
+            msg = _('User is not staff member.')
+            raise serializers.ValidationError(msg)
```

Comment:
Cette vérification est un peu complexe. Je suggère plutôt :

    if user.is_superuser or user.is_staff:
        return attrs
    raise serializers.ValidationError(_('User is not staff member.'))